configs/efficientnet/README.md

```diff
@@ -0,0 +1,39 @@
+# EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks
+
+## Introduction
+
```

Comment:
Here we need a tag `<!-- [ALGORITHM] -->` to indicate the type of the algorithms, see example of swin
https://raw.githubusercontent.com/open-mmlab/mmdetection/master/configs/swin/README.md